src/components/Pages/Guide/_Guide.scss

```diff
@@ -79,6 +79,14 @@ $max-guide-tablet-width: $coa-medium-screen;
   overflow-y: scroll;
   margin-top: $sticky-guide-top;
   top: 0px;
+
+  // Hide the nav scrollbar in Chrome/Safari
```

Comment:
Do we know which one of these applies to Firefox? I tested quickly and didn't see a scrollbar on the guide TOC so I think we're good. 

Also can we update the note to say something like:
" Hides the nav scrollbar for guide page (used for the Guide Menu) "

Better yet could we add a class like coa-GuidePage__content-container--hide-scrollbar so its a bit more self-documenting?

Reply:
It's in the `.coa-GuideMenu__desktop-container` class, I'm not sure how (used for guide menu) makes that more clear. I feel like adding another class for it might get a little too verbose for my liking.

That's interesting that it's working in firefox, I forgot to check that yesterday. I saw this: https://stackoverflow.com/a/53787170 but haven't actually put that in there yet. I'll poke at it a bit

Reply:
Yeah! I saw that too and just figured there was a reason why it was the way it was. 

What I was thinking was ```.coa-GuideMenu__desktop-container--hide-scrollbar``` would follow BEM convention and also be more self-documenting, so we probably wouldn't need a verbose comment as well. 

But I'm far from an expert on BEM or css stuff, so I defer to the team overall.

Reply:
I'm a fan of having explicit comments within classes. That isn't something to be avoided. `.coa-GuideMenu__desktop-container` is verbose enough to let us know that this class is for containers on desktop on a guide page. If a style only needs to be added to one class, it should not be separated out into separate classes. Even `.coa-GuideMenu__desktop-container--hide-scrollbar` isn't self-documenting enough. Following that logic, we'd have to break it down into `.coa-GuideMenu__desktop-container--hide-scrollbar-chrome-and-safari`. 

Comments in css are useful, it's good to know why styles are added. And reducing complexity and bloat by keeping things in as few classes as possible is preferred.